jsonschema/catalog.json

```diff
@@ -7,7 +7,7 @@
       "product": "Cloud Audit Logs",
       "name": "LogEntryData",
       "description": "Generic log entry, used as a wrapper for Cloud Audit Logs in events.\n This is copied from\n https://github.com/googleapis/googleapis/blob/master/google/logging/v2/log_entry.proto\n and adapted appropriately.",
-      "datatype": "google.events.cloud.audit.v1.LogEntryData",
```

Comment:
I thought `datatype` was meant to be "fully qualified protobuf message name"? It's only the CloudEvent type that doesn't have ".events".